buildSrc/src/main/kotlin/plugins/docs.gradle.kts

```diff
@@ -0,0 +1,19 @@
+import org.jetbrains.dokka.gradle.DokkaMultiModuleTask
+import org.jetbrains.dokka.gradle.DokkaTask
+
+plugins {
+    id("org.jetbrains.dokka")
+}
+
+tasks {
+    withType<DokkaTask>() {
+        dokkaSourceSets.configureEach {
+            includes.from(listOf("README.md"))
+        }
+    }
+
+    withType<DokkaMultiModuleTask>().configureEach {
+        val dokkaDir = buildDir.resolve("dokka")
+        outputDirectory.set(dokkaDir)
+    }
```

Comment:
Is that mandatory? I mean, there's already a default value, right?
Maybe you did that so I can change it right here later on when I setup the website?

Reply:
I have that value since years, it is not necessary anymore, removed.